src/SignalR/clients/ts/signalr-protocol-msgpack/package.json.bak

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Revert all the changes you've done to packages files. Make sure not to add them to the future commits either.